Kubernetes/omsagent.yaml

```diff
@@ -571,6 +571,103 @@ spec:
           configMap:
             name: container-azm-ms-agentconfig
             optional: true
+---
+  apiVersion: apps/v1
+  kind: DaemonSet
+  metadata:
+   name: omsagent-win
+   namespace: kube-system
+   labels:
+    component: oms-agent
+    tier: node
+  spec:
+   updateStrategy:
+    type: RollingUpdate
+   selector:
+    matchLabels:
+      component: oms-agent
```

Comment:
When create a PR into AKS RP : talk with Robbie/Anders about component

Reply:
Add this in the AKS RP changes task

Reply:
AKS Team cpu monitoring agent will need to check for 'omsagent' + 'omsagent-win'.